### DISCUSSION TITLE
[MONAILabel] json.decoder.JSONDecodeError

### DISCUSSION THREAD

**Discussion by ulphypro** [Project-MONAI/MONAILabel#1581] [Q&A] [answered]
Dear, all members

I have the question.

I'm currently training  after segmenting 347 medical imaging data using 3D-Slicer based on MONAILabel.

But it suddenly occurs some error as follows when I trained after segmenting 172th medical imaging data.

Please, let me know how to solve this problem.

![123](https://github.com/Project-MONAI/MONAILabel/assets/67679322/44dab3d4-8db5-4a36-a2ff-c78cec23344f)

**Comment by SachidanandAlle** [comment, marked as answer]
in your studies folder dataset.json is created by monailabel.. try to vim and see if the file is a valid json..
if it's corrupted, you can delete it and start the monailabel server.. it will try to construct again based on the images in studies folder and labels from labels/{final} folder

**Reply by ulphypro** [reply]
Dear @SachidanandAlle 


Thank you for answering my question.

I reinstalled monailabel module and monailabel package, it runs well.
